lib/qwerty/text.rb

```diff
@@ -0,0 +1,12 @@
+module Qwerty
+  class Text < Ruote::Participant
+    def on_workitem
+      workitem.fields[:text] = {
+         :format => 'text',
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.